src/Components/WebAssembly/Build/test/Microsoft.AspNetCore.Components.WebAssembly.Build.Tests.csproj

```diff
@@ -22,20 +22,16 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <Reference Include="Microsoft.AspNetCore.Blazor.Build" />
+    <Reference Include="Microsoft.AspNetCore.Components.WebAssembly.Build" />
     <Reference Include="Microsoft.AspNetCore.Blazor.Mono" />
-    <Reference Include="Microsoft.AspNetCore.Mvc.Razor.Extensions" />
```

Comment:
There's a few tests deleted from the Blazor.Build project. However they were testing the compiler's codegen, which isn't a responsibility of this package. I noticed this because they were also relying on an internal API in the Components package that was visible as a result of https://github.com/dotnet/aspnetcore/blob/master/src/Components/Components/src/Properties/AssemblyInfo.cs#L3

Reply:
It's fine